### ISSUE TITLE
Shade of Aran is bugged

### ISSUE THREAD

**Issue by KarlDenXII** [Atlantiss/NetherwingBugtracker#5312]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 
Shade of Aran is not scripted properly.

**Current behaviour**: 
Currently Shade of Aran spams his spells and does not use his melee attacks, instead chaining his casts. Also his Blizzard is too large.

**Expected behaviour**: 
Should use melee attacks, Blizzard should be smaller.

Proof: https://www.youtube.com/watch?v=l0DB4lzAttw

**Server Revision**: 
2868

**Comment by Kristoferhh** [comment]
I've seen him melee us in the face sometimes. I thought _that_ was a bug lol.

**Comment by Wolffenstein** [comment]
He isn't supposed to melee unless he's locked out of all of his spell schools.

**Comment by Drayo** [comment]
He is behaving pretty much exactly as he did on retail.

**Comment by towneh** [comment]
Melee aside, I would still consider the Blizzard spell mechanic in this issue thread however. 
This [video](https://www.youtube.com/watch?v=QJOWLwr8xlE) from 17th Feb 2007 seems to highlight a version of Blizzard that travels a lot slower, has less spell effects and appears to be pushed out closer to the alcoves away from the center.
It feels like the version of Blizzard in the video is there mostly to stop people just mindlessly casting from within an alcove safe spot.

**Comment by Drayo** [comment]
Pretty sure this look so different because players (esp ones that recorded) had their spell details on the lowest setting.

I reported potential safespots from the blizzard before (at the door), but it was said the blizzard movement came from sniffs and considering that the safespots are not reliable, using them would be hard since you would just die if it just happens to not work in that moment.

Proving that these safespots did not exist back in the day is impossible, and I found no videos of people using any either, so any changes regarding this would be based on a guess.
Thus I understand that the devs use the only proper source they have and leave it at that.
